bench/workgen/workgen.h

```diff
@@ -341,9 +341,9 @@ struct Operation {
 // properties are prevented, only existing properties can be set.
 struct ThreadOptions {
     std::string name;
-    std::string session_config;
     double throttle;
     double throttle_burst;
+    std::string session_config;
```

Comment:
The data types should be grouped together by size so `session_config` should be under `name`

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9970/commits/aae1e5a63d9af7235108e5bf2350b6c74001c6ce